<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ymin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ymax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xmin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xmax</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Xmin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Xmax">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Ymin">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Ymax">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
